Utils/Hash/libcrc/lib_crc.c

```diff
@@ -76,7 +76,7 @@
     *   static unsigned ... crc_tab...[]                                *
     *                                                                   *
     *   The algorithms use tables with precalculated  values.  This     *
-    *   speeds  up  the calculation dramaticaly. The first time the     *
+    *   speeds  up  the calculation dramatically. The first time the     *
```

Comment:
I didn't try to get the `*`s to line up. I can do that when I respond to a first round of review comments

Reply:
(Changed locally)